step.yml

```diff
@@ -1,26 +1,57 @@
 ---
-title: Artifact pull
-summary: Step to pull artifacts of a pipeline
+title: Pull Pipeline intermediate files
+summary: Pull intermediate files generated by the Pipeline Builds.
 description: |-
-  The step downloads build artifacts of a pipeline to a local folder.
+  The step downloads intermediate files (build artifacts generated by the pipeline workflows,
+  that are intended to be shared with subsequent workflows of the pipeline) to a local folder.
 
-  By default, all artifacts generated by any workflow of the pipeline are downloaded. This can be limited
-  by setting the `artifact_sources` input variable.
+  These artifacts can be uploaded by the [Deploy to Bitrise.io step](https://www.bitrise.io/integrations/steps/deploy-to-bitrise-io),
+  via specifying the *Files to share between pipeline stages* input.
+
+  Pipeline intermediate files have an environment variable key assigned,
+  after downloading the given artifact, an environment variable (with the assigned key) will point to the file's local path.
+
+  Directories specified in *Files to share between pipeline stages* input will be archived and uploaded as a single file.
+  After downloading, they are extracted, and the specified environment variable will point to the directory's local path.
+
+  By default, all artifacts generated by any workflow of the pipeline are downloaded.
+  This can be limited by setting the *Artifact source* input.
 
   Please note that this step is designed to be executed on the CI only.
 
 website: https://github.com/bitrise-steplib/steps-artifact-pull
 source_code_url: https://github.com/bitrise-steplib/steps-artifact-pull
 support_url: https://github.com/bitrise-steplib/steps-artifact-pull/issues
+
+run_if: .IsCI
+
 type_tags:
 - utility
-is_always_run: false
-is_skippable: false
+
 toolkit:
   go:
     package_name: github.com/bitrise-steplib/steps-artifact-pull
 
 inputs:
+- artifact_sources: .*
+  opts:
+    title: Artifact source
+    summary: |-
+      A comma (`,`) separated list of the stage and workflow
+      paths, which generated artifacts to download.
```

Comment:
A comma (`,`) separated list of the Stage and Workflow paths, used to specify which generated artifacts to download.